### ISSUE TITLE
JSXEmptyExpression not supported

### ISSUE THREAD

**Issue by k2snowman69** [snowcoders/sortier#497]
# Language
Typescript

# Sample input source code
https://github.com/piotrwitek/react-redux-typescript-guide/blob/master/playground/src/components/mouse-provider.tsx
```
  render() {
    return (
      <div style={{ height: '100%' }} onMouseMove={this.handleMouseMove}>
        {/*
          Instead of providing a static representation of what <Mouse> renders,
          use the `render` prop to dynamically determine what to render.
        */}
        {this.props.render(this.state)}
      </div>
    );
}
```

# Expected output
```
  render() {
    return (
      <div onMouseMove={this.handleMouseMove} style={{ height: '100%' }}>
        {/*
          Instead of providing a static representation of what <Mouse> renders,
          use the `render` prop to dynamically determine what to render.
        */}
        {this.props.render(this.state)}
      </div>
    );
}
```

# Actual output (or error message)
If help mode is turned on we get:
```
 - JSXEmptyExpression - {"line":25,"column":9} - {"line":28,"column":10}
/*
          Instead of providing a static representation of what <Mouse> renders,
          use the `render` prop to dynamically determine what to render.
        */
```

**Comment by k2snowman69** [comment]
Will release with 2.6.3 or greater

**Comment by k2snowman69** [comment]
Released with 3.0.0
